pkg/helm/actions/template.go

```diff
@@ -0,0 +1,105 @@
+package actions
```

Comment:
is this used anywhere?

Reply:
from GetImageReferences  in checks/helm.go.

Reply:
do we really need to have it in a separate package? How about moving it under checks/helm.go?